src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+public class SetEnvironmentVariable
```

Comment:
Ok.  Don't block merging this based on that, just something to consider adding.  Might also be easier to add (and still fit) to the Process test suite rather than here.